.gitignore

```diff
@@ -0,0 +1,52 @@
+# Prerequisites
```

Comment:
Please also add the following to `.gitignore`:

- `Makefile` (it's auto-generated)
- `autom4te.cache/*`
- `config.log`
- `config.status`
- `libtool`
- `ltmain.sh`
- `*.bak`
- `m4/libtool.m4`
- Any other files under `m4` that are auto-generated